packages/gasket-plugin-intl/README.md

```diff
@@ -53,7 +53,7 @@ required. However, these options exist to customize an app's setup.
   If set to `true`, the app will use the `defaultPath` as the static endpoint
   path. This option can also be set to a string, to be used as the static
   endpoint path.
-- `modules` - (boolean|object) Enable locale files collation from node modules.
+- `modules` - (boolean|object|string[]) Enable locale files collation from node modules.
```

Comment:
Could you also change the type of modules [in the TypeScript definition](https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-intl/lib/index.d.ts#L16) as well?

Reply:
Maybe you started reviewing before [this commit](https://github.com/godaddy/gasket/pull/676/commits/e9d010005dc93f25de0d5d0c47b1868fe21cf301)? - Should be all set.